tests/test_functional.py

```diff
@@ -35,15 +35,55 @@ def test_home_page(self, testapp, db):
         assert "Evidence dárců ČČK Frýdek-Místek" in res
         assert res.status_code == 200
 
-    def test_pages_401(self, testapp, db):
+
+class TestErrorInterface:
+    """Test 404 <span class="x x-first x-last">and 401 </span>pages"""
```

Comment:
Why do we have HTML code in the docstring?

Reply:
That was in your suggestion. I've supposed that it is your intention.. I'll fix it